### ISSUE TITLE
[AVT]Keyboard focus indicator can not be visually located on Listbox in service create page

### ISSUE THREAD

**Issue by cmqchen** [carbon-design-system/carbon#4445]
Describe the issue
Keyboard focus indicator can not be visually located on Listbox in service create page.
It is disabled because there is only one entry and this acts like a read-only field, but users can use keyboard to focus on this component, so please make the focus indicators visible.
![image](https://user-images.githubusercontent.com/56956281/67467888-32319a80-f67c-11e9-8c07-2c937539a65f.png)

IBM CI 162 Checkpoint Violation: IBM 2.4.7 Focus Visible
link: https://www.ibm.com/able/guidelines/ci162/focus_visible.html

To Reproduce
Login to IBM Cloud
Click Catalog -> Compute
Click Hyper Protect Virtual Servers

Expected behavior
Keyboard focus indicator can be visually located so that a person knows which element has the keyboard focus. All focus indicators should be highly visible and distinguishable.

Affected Environments
[https://test.cloud.ibm.com/catalog/services/hyper-protect-virtual-servers] Staging (test.cloud.ibm.com)

**Desktop **
OS: [Windows]
Browser [Chrome]

Screenshots
![listbox](https://user-images.githubusercontent.com/56956281/67467651-b7687f80-f67b-11e9-9d93-f0432fccab57.png)

**Comment by asudoh** [comment]
Hi 👋 thank you for reporting! Would you want to isolate the problem more, e.g. wrt what Carbon component you see the problem with? Best with a reduced case based on https://codesandbox.io/s/github/carbon-design-system/carbon/tree/master/packages/components/examples/codesandbox or https://codesandbox.io/s/github/carbon-design-system/carbon/tree/master/packages/react/examples/codesandbox. Thanks!

**Comment by cmqchen** [comment]
It's a dropdown component. 
When there is only one item in dropdown list, it's read only. Users can not see the focus indicator located on this component by using keyboard.
![image](https://user-images.githubusercontent.com/56956281/67549887-d4fb1f00-f737-11e9-9dd3-35c8785b19d9.png)
When there are several items in dropdown list, users can see the focus indicator located on this component by using keyboard. This is normal.
![image](https://user-images.githubusercontent.com/56956281/67549985-0ecc2580-f738-11e9-855b-1e83d295297b.png)
![image](https://user-images.githubusercontent.com/56956281/67551054-8dc25d80-f73a-11e9-840f-d99276fc87ce.png)

**Comment by asudoh** [comment]
Given the screenshot you seem to be using `<DropdownV2>` in `v6`, but cannot reproduce the problem in my local dev env even though I limited the dropdown items to one. So I recommend you create a reduced case based on https://codesandbox.io/s/github/carbon-design-system/carbon/tree/master/packages/react/examples/codesandbox.

**Comment by cmqchen** [comment]
This defect occurred in the IBM Cloud, the Cloud team will make some changes on this component to meet the AVT rules, so I close the defect first.

**Comment by cmqchen** [comment]
Sorry, this defect has not been verified, may also need your help, so reopen this defect.

**Comment by dakahn** [comment]
Also can't reproduce this error in our v9 Dropdown example. If this issue persists and you can provide a reduced test case we can debug against i'll reopen.
